blurring_as_a_service/inference_pipeline/components/detect_and_blur_sensitive_data.py

```diff
@@ -69,15 +69,13 @@ def detect_and_blur_sensitive_data(
     with open(f"{yolo_yaml_path}/pano.yaml", "w") as outfile:
         yaml.dump(data, outfile, default_flow_style=False)
     cuda_device = torch.cuda.current_device()
-    model_parameters = settings["inference_pipeline"]["model_parameters"]
+    model_parameters = json.loads(model_parameters_json)
```

Comment:
where is this json?

Reply:
This is the config json that is in the root of the repo. With subription name etc

Reply:
but that only contains the subscription id, the rg and the workspace, right?

Reply:
What is the reason to not use the settings file anymore?

Reply:
pls, see my other comment

Reply:
Sorry if I'm misunderstanding, but why can't we do 
`model_parameters = BlurringAsAServiceSettings.get_settings()["inference_pipeline"]['customers'][0]['model_parameters']` ?

Reply:
Chris: Try to use customer as key (omdat we customer str al doorsturen)

Reply:
I think we should set one setting configuration per customer. Because more things can change between customers, including model versions, parameters, and things like that. So if you set one settings file per customer becomes easy to track and store all the parameters keeping the code as generic as possible.